invenio_oaiserver/percolator.py

```diff
@@ -43,19 +46,19 @@ def _create_percolator_mapping(index, doc_type, mapping_path=None):
     percolator_index = _build_percolator_index_name(index)
     if ES_VERSION[0] in (5, 6):
         current_search_client.indices.put_mapping(
-            index=index, doc_type=doc_type,
-            body=PERCOLATOR_MAPPING)
+            index=index, doc_type=doc_type, body=PERCOLATOR_MAPPING
```

Comment:
we should try to avoid styling changes... in any case it makes me thing... If we will release this as par of a new minor version (i.e. not getting in invenio < 3.4)... can we remove Elasticsearch 5?

Reply:
not worth spending the time now.